resources/views/components/turnstile.blade.php

```diff
@@ -1,9 +1,13 @@
+@php
```

Comment:
Since this is a Blade component, it would make more sense to have a prop called `id` with a default value, so that people can pass their own IDs through too.